index.html

```diff
@@ -8,340 +8,305 @@
+<body>
+    <section id="detail1">
+      <button type="button" id="modal-close">X</button>
+      <div id="detail"></div>
+      <!-- <section class="title">
+        <h2>Multi-Post stroies</h2>
+      </section>
+  
+      <figure><img src="/images/portfolio.png" alt="Project Image"></figure>
+      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+        when an unknown printer took a galley of type and scrambled it 1960s with the 
+        releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, 
+        when an unknown printer took a galley of type veris lapoa todoe.</p>
+      
+      <ul class="tags">
+        <li>html</li>
+        <li>Ruby on rails</li>
+        <li>CSS</li>
       </ul>
-    </nav>
-  </header>  
-
-  <main>
-    <section id="headline">
-      <div class="ellipse_18"></div>
-      <article class="headlines">
-        <h2> Owusu Dennis </h2>
-        <h3> Software Developer </h3>
-      </article>
-
-      <article class="brand">
-        <p>
-          I can help you build a product, 
-          feature or website Look through some of my 
-          work and experience! If you like what you 
-          see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-      </article>
-      <hr>
-    </section>  
-    
-    <section id="frame_container">
-      <article class="frame">
-        <section class="languages">
-          <h3>Languages</h3>
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </section>
-
-        <div class="back"></div>
-        
-        <section class="frameworks">
-          <h3>Frameworks</h3>
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </section>
-      </article>
-
-      <article class="skills">
-        <h3>Skills</h3>
-        <ul>
-          <li>Codekit</li>
-          <li>Github</li>
-          <li>Codepen</li>
-          <li>GitLab</li>
-          <li>Terminal</li>
-        </ul>
-      </article>
-    </section>
-
-    <hr>
-    
-    <section id="social_mediaWork">
-      <section class="work_first">
-        <a href="#"><img src="/images/Element_Right.png" alt=""></a>
+  
+      <section id="button-group">
+        <button type="button" class="demo-button">
+          <span>See Live</span> 
+          <span><img id="live-icon" src="/images/live.png" alt="live icon"></span>
+        </button>
+        <button type="button" class="source-button">
+          <span>See Source</span> 
+          <span><img class="github-icon" src="/images/one.png" alt="live icon"></span>
+        </button>  -->
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻